### PR TITLE
chore(deps): Update dependencies and plugins using properties

### PR DESCRIPTION
**Pull request by reneleonhardt**
Introduce properties for dependency and plugin versions, and update them to the latest versions.

- animal-sniffer-maven-plugin 1.2 to 1.23
- junit 5.9.2 to 5.9.3
- jacoco-maven-plugin 0.8.8 to 0.8.10
- maven-wrapper 3.1.1 to 3.2.0
- maven 3.9.0 to 3.9.2

### CODE REVIEW

**Comment by pjfanning** [comment]
I'm afraid we have dependabot and central version management in https://github.com/FasterXML/oss-parent

**Comment by cowtowncoder** [comment]
That, plus plug-in versions come from combination of parent poms (oss-parent, jackson-parent, jackson-base, jackson-bom). They are not to be specified locally except temporarily.

File: pom.xml [hunk 1]
```diff
@@ -53,6 +53,14 @@ tools.jackson.core.*;version=${project.version}
 
     <!-- for Reproducible Builds -->
     <project.build.outputTimestamp>2022-11-27T00:00:00Z</project.build.outputTimestamp>
+
+    <!-- dependency and plugin versions -->
```
Comment (cowtowncoder) [inline pom.xml:57]: These are to be upgraded in parent pom:

https://github.com/FasterXML/oss-parent/

so could you please remove?

File: pom.xml [hunk 2]
```diff
@@ -275,7 +285,7 @@ tools.jackson.core.*;version=${project.version}
     <dependency>
       <groupId>ch.randelshofer</groupId>
       <artifactId>fastdoubleparser</artifactId>
-      <version>0.9.0</version>
+      <version>${fastdoubleparser.version}</version>
```
Comment (cowtowncoder) [inline pom.xml:288]: No point in add property just for sake of adding one; will never be overridden or reused.

**Comment by cowtowncoder** [comment]
Thank you for submitting this; as per my notes, closing, will apply mvnw changes separately however & upgrade animal sniffer.

**Pull request closed without merging**